buildscripts/build.cmd

```diff
@@ -16,9 +16,11 @@ REM ****************************************************************************
 
 IF NOT EXIST %~dp0..\Settings.proj GOTO msbuild_not_configured
 
-REM Set Framework version based on passed in parameter
 IF "%1" == "" goto no_nothing
 
+REM Set Framework version based on passed in parameter
```

Comment:
This comment seems to relate to the lines below referring to NET40. This script is so old that I'm not sure if everything is still used.